nalu-processor/pom.xml

```diff
@@ -267,6 +267,30 @@
           </execution>
         </executions>
       </plugin>
```

Comment:
Have to re-state the plugin, because we have to change the plugin execution order due to the shade plugin

Reply:
Ok